pkg/snowflake/grant_test.go

```diff
@@ -98,7 +98,7 @@ func TestWarehouseGrant(t *testing.T) {
 
 }
 
-func TestAccountGrant(t *testing.T) {
+func TestAccountGrant(t *testing.T) { // lintignore:AT003
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT003: acceptance test function name should include underscore

Reply:
I am trying to get this ignored